docs/help/continuous-integration/github-actions.md

```diff
@@ -30,6 +30,10 @@ on:
     branches:
       - main
 
+permissions:
+  contents: read
+  pull-requests: write
+
```

Comment:
You don't need specific permissions for the `deploy` action only

Reply:
```suggestion

```

Reply:
Oh yes sure, good catch thanks!

Changed (even if _qui peut le plus peut le moins_ 🇫🇷 )